src/utils.js

```diff
@@ -38,10 +38,14 @@ const Utils = {
     return results
   },
 
-  log: (message) => {
+  log: (message, extra) => {
     if (options.debug) {
       // eslint-disable-next-line no-console
-      console.log(message)
+      if (!!extra) {
+        console.log(`[Alephbet] ${message}`, extra)
```

Comment:
@joker-777 I found https://developer.mozilla.org/en-US/docs/Web/API/Console/log#logging_objects 

>  Don't use console.log(obj), use console.log(JSON.parse(JSON.stringify(obj))). 
>
>  This way you are sure you are seeing the value of obj at the moment you log it. Otherwise, many browsers provide a live view that constantly updates as values change. This may not be what you want. 

So I will update it.